tests/js-tests/project/proj.android/app/build.gradle

```diff
@@ -32,12 +42,17 @@ android {
     }
 
     externalNativeBuild {
-        ndkBuild {
-            if (!project.hasProperty("PROP_NDK_MODE") || PROP_NDK_MODE.compareTo('none') != 0) {
+        if (!project.hasProperty("PROP_NDK_MODE") || PROP_NDK_MODE == 'ndk') {
```

Comment:
`PROP_NDK_MODE == 'ndk'` where set PROP_NDK_MODE to ndk?

Reply:
in file __gradle.properties__, it set `PROP_NDK_MODE `, control the native compile method by comment it and uncomment it, another way you can set value to `ndk` or `cmake` to control
```
# android cmake support
# uncomment it, native code will build by cmake
# keep comment, native code will build by ndkBuild
# PROP_NDK_MODE=cmake
```